### PR TITLE
wayland: create bunsenlabs-session-wayland.1 manpage

### PR DESCRIPTION
**Pull request by 01micko**
populate wayland/skel with autostart and some configs add/fix debian/bunsen-config-wayland.* files

### CODE REVIEW

**Comment by 01micko** [comment]
Since this is 100% wayland related I'll merge.

**Comment by johnraff** [comment]
Thanks!

File: wayland/bunsenlabs-session-wayland.1 [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+arguments.
+.PP
+\fBbunsenlabs-session-wayland\fR is set as the Debian alternative
+"wayland-session-manager" so it will be run by default in a Debian wayland
+session, and a .desktop file is installed to /usr/share/wayland-sessions
+so it can be started by a display manager like LightDM.
+.PP
+On log in, \fBbunsenlabs-session-wayland\fR will run the
+~/.config/bunsen/autostart-wayland script if it exists, and will copy in the
+default file from /usr/share/bunsen/skel/.config/bunsen/autostart otherwise. You may
```
Comment (johnraff) [inline wayland/bunsenlabs-session-wayland.1:25]: default file is /usr/share/bunsen/skel/.config/bunsen/ autostart-wayland

File: wayland/skel/.config/bunsen/autostart-wayland [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+## BunsenLabs autostart
+## ====================
+## When you login to your BunsenLabs session, this autostart script
+## will be executed to launch any applications
+## you want to run at startup.
+##
+## NB, like openbox's and labwc's autostart script, this uses the default shell,
+## so "bashisms" are not supported.
+##
+## (Environment variables are set in ~/.config/bunsen/environment.)
```
Comment (johnraff) [inline wayland/skel/.config/bunsen/autostart-wayland:15]: Environment variables are set in ~/.config/bunsen/environment-wayland

**Comment by johnraff** [comment]
Two small tweaks though.

File: wayland/bunsenlabs-session-wayland.1 (hunk 1 above)
Comment (01micko) [reply wayland/bunsenlabs-session-wayland.1:25]: missed that one, fix in next commit

File: wayland/skel/.config/bunsen/autostart-wayland (hunk 2 above)
Comment (01micko) [reply wayland/skel/.config/bunsen/autostart-wayland:15]: ..and that one

**Pull request merged**